sc3-sc3-calc-biology.R

```diff
@@ -45,14 +45,13 @@ option_list = list(
   )
 )
 
-opt <- wsc_parse_args(option_list, mandatory = c('input_object_file', 'output_object_file', 'output_dir'))
+opt <- wsc_parse_args(option_list, mandatory = c('input_object_file', 'output_object_file', 'output_text_file'))
```

Comment:
LGTM! One small ask, could you remove `output_text_file` from list of mandatory options, as the information is already in the object and the text file can be large given the length of Ks.

Reply:
Okay, done- look good?